metricsbp/config.go

```diff
@@ -53,6 +53,12 @@ type Config struct {
 	//
 	// Optional, defaults to false.
 	RunSysStats bool `yaml:"runSysStats"`
+
+	// ReportServerConnectionCount indicates that you want to publish
+	// a counter for the number of clients connected to the server
+	//
+	// Optional, defaults to false.
+	ReportServerConnectionCount bool `yaml:"ReportServerConnectionCount"`
```

Comment:
the other yaml things are `lowerCamelCase` which while I don't love, you want to at least be consistent :D